tools/validators/instance_validator/validate/telemetry.py

```diff
@@ -30,6 +30,7 @@
 PRESENT_VALUE = 'present_value'
 PARTIAL_UPDATE = 'partial_update'
 
+# pylint: disable=consider-using-f-string
```

Comment:
Same here. The reason you wanted to use `.format()` was for keyed values but you're not really using that in this module. Probably worth switching to f-string literals.